### PR TITLE
[PNDM in LDM pipeline] use inspect in pipeline instead of unused kwargs

### PR DESCRIPTION
**Pull request by patil-suraj**
This PR updates the `LDMTextToImagePipeline` and `LDMPipeline` to determine wether the `scheduler.step` accepts  the `eta` arg using `inspect.signature`. And removes the unused `kwargs` from `PNDMScheduler`

### CODE REVIEW

File: src/diffusers/schedulers/scheduling_pndm.py [hunk 1]
```diff
@@ -116,7 +116,6 @@ def step(
         model_output: Union[torch.FloatTensor, np.ndarray],
         timestep: int,
         sample: Union[torch.FloatTensor, np.ndarray],
-        **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_pndm.py:119]: <3
